Debugger.py

```diff
@@ -109,6 +118,8 @@ def validatePaths(argsWithPaths):
         usableArgs.software = False
         usableArgs.address = FIRMWARE_LOAD_ADDRESS
 
+    if usableArgs.breakpoint is not None:
+        breakpointFile = usableArgs.input.split(".")[0] + ".bp"
```

Comment:
A path check should be done on this to make sure it's valid. Something similar to what is done on the symbols file. Note that the validatePaths function might need to be adjusted (choice of variable name and other things).